packages/ui/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { cx } from "class-variance-authority";
+import { TooltipContentProps } from "@radix-ui/react-tooltip";
+
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "./Tooltip";
+import { Button } from "../Button";
+
+const meta = {
+  title: "Components/Tooltip Content",
+  component: TooltipContent,
+  tags: ["autodocs"],
+  argTypes: {
+    align: {
+      control: "select",
+      options: ["start", "center", "end"],
+      defaultValue: "top",
+      description:
+        "Represents the preferred alignment of the content relative to the trigger.",
+    },
+    side: {
+      control: "select",
+      options: ["top", "right", "bottom", "left"],
+      defaultValue: "top",
+      description: "Sets the content alignment relative to the trigger.",
+    },
+    sideOffset: {
+      control: "number",
+      defaultValue: 0,
+      description:
+        "Represents the distance in pixels between the content and the trigger",
+    },
+  },
+} satisfies Meta<typeof TooltipContent>;
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+const basicTooltipPropList: Array<TooltipContentProps> = [
+  { align: "end", side: "bottom" },
+  { align: "center", side: "bottom" },
+  { align: "start", side: "bottom" },
+  { align: "end", side: "right" },
+  { align: "center", side: "top" },
+  { align: "start", side: "left" },
+  { align: "end", side: "top" },
+  { align: "center", side: "top" },
+  { align: "start", side: "top" },
+];
+export const Basic: Story = {
+  render: (args) => (
+    <div
+      className={cx(
+        "w-full h-[95vh] grid grid-cols-3 grid-rows-3",
+        // Second column
+        "[&>*:nth-child(3n+2)]:justify-center",
+        // Third column
+        "[&>*:nth-child(3n)]:justify-end",
+        // First row
+        "[&>:nth-child(-n+3)]:items-start",
+        // Second row
+        "[&>:nth-child(n+4):nth-child(-n+6)]:items-center",
+        // Third row
+        "[&>:nth-child(n+7)]:items-end"
```

Comment:
this seems too complicated or a simple grid, why not just use the `grid-cols-3` and the gap property to give spacing between them or something like this?

Reply:
Because my goal was to place all the examples along the container borders, so we can check if that the tooltip never "disappears" from the contained view (it should go outside the screen)